src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -40,4 +44,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 
   response.getOutputStream().println(json);
  }
+
+ @Override
+ public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+   UserService userService = UserServiceFactory.getUserService();
+   if (!userService.isUserLoggedIn()) {
+     response.sendRedirect("/index.html");
+     return;
+   }
+
+   String user = userService.getCurrentUser().getEmail();
```

Comment:
Nit: prefer the variable name userEmail, since it is more descriptive

Reply:
ok, I'll fix it!